.github/workflows/closed-issue-message.yaml

```diff
@@ -0,0 +1,21 @@
+name: Add comment
```

Comment:
Not sure how it is configurable, but if 2 peoples have a discussion on a closed issue, would they have this message posted every-time one of them answers?

Reply:
No... it only runs on the 'issue.closed' event

Reply:
Oh OK, so on the other hand it has 2 issues:

1. It does not fix the problem for already closed issue => this one is problematic but maybe we an run a script to add it on all the closed issues 
2. It could be missed if the issue is closed for a long time and has a lot of answers since => that one is probably negligible IMHO

Reply:
> 1. It does not fix the problem for already closed issue => this one is problematic but maybe we an run a script to add it on all the closed issues

Yes, I can build a simple node script we can run once that adds this comment to all already closed issues. (This should be done right before we merge this one, so we do not miss any issues)

> 2. It could be missed if the issue is closed for a long time and has a lot of answers since => that one is probably negligible IMHO

Yes, thats right. For already closed issues this will be added as last comment. But you are right. This is not that important.

Reply:
So. I just played around with the REST API and apparently there are around 10.643 closed issues. 😅 

I could run an "add comment" with that script and this message so we are handling every issue the same.

WDYT?

Reply:
cc @joserodolfofreitas

Reply:
> So. I just played around with the REST API and apparently there are around 10.643 closed issues.

I see value in leaving the message on every issue, but I also think we can skip that considering how often this actually happens.